cobalt/updater/network_fetcher.cc

```diff
@@ -158,9 +188,15 @@ void NetworkFetcher::OnURLFetchComplete(const net::URLFetcher* source) {
   const int response_code = source->GetResponseCode();
   if (url_fetcher_type_ == kUrlFetcherTypePostRequest) {
     OnPostRequestComplete(source, status.error());
+#if !defined(IN_MEMORY_UPDATES)
```

Comment:
Could we try to always use positive conditionals?

Reply:
I think in this case it's more clear to use the positive as you suggested. I also changed it in a few other places in the PR.

I assume you're not including the situation where we *just* want to remove some code when !FOO, and there's nothing to add when FOO.

Also, I do think there are some cases where it's more clear to use the negative even when there is something to include when FOO. A good example is an a API declaration in a header where there are significant function comments. Here, I think it's easier to read when it's displayed like:

```
#if !defined(FOO)
// Comments here talking about the behaviors of the functions, use of the params, etc.
void some_function(int x, int y);
#else
// Brief comment about how this version of the function is different.
void some_function(int x, int y, string z);
#endif
```

Reply:
Yep I just meant for if/elses and makes sense for comments like you said. I still see a few cases like https://github.com/youtube/cobalt/pull/849/files?diff=split#diff-d0722d8c7ed7327d8a3c8ed3acb4669357b9a1e450d2a5ebf810f608d008501eR112 where I'm not sure why it would be kept negative first though?

Reply:
Cool, think the rest are fixed now.

Reply:
yep everything looks good thanks!